shell/platform/fuchsia/runtime/dart/utils/build_info.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_FUCHSIA_RUNTIME_DART_UTILS_BUILD_INFO_H_
+#define FLUTTER_SHELL_PLATFORM_FUCHSIA_RUNTIME_DART_UTILS_BUILD_INFO_H_
+
+#include "root_inspect_node.h"
+
+namespace dart_utils {
+
+class BuildInfo {
+ public:
+  static const char* FlutterEngineGitRevision();
```

Comment:
If it is easy to grab can you also add the fuchsia SDK version as well? If not we can add it later.

Reply:
yup! I'll add that as `fuchsia_sdk_version`.